requirements.txt

```diff
@@ -0,0 +1,14 @@
+certifi==2019.6.16
+chardet==3.0.4
+Click==7.0
+Flask==1.1.1
+gitdb2==2.0.5
+GitPython==2.1.11
+idna==2.8
+itsdangerous==1.1.0
+Jinja2==2.10.1
+MarkupSafe==1.1.1
+requests==2.22.0
+smmap2==2.0.5
+urllib3==1.25.3
+Werkzeug==0.15.5
```

Comment:
Missing dependencies in requirements.txt:
- Flask-Cors
- gogs_client
- cryptography

Reply:
Updated the requirements in latest push.

Reply:
I see a commit that says `Updated requirements`, but there's no changes to the requirements file in there.